### PR TITLE
Update nw-metallb-installing-operator-cli.adoc

### PR DESCRIPTION
**Pull request by Rupesh-git-eng**
Operator installation output is sometimes not immediately available. It gives empty output i.e 

$ oc get clusterserviceversion -n metallb-system -o custom-columns=Name:.metadata.name,Phase:.status.phase
Name   Phase
$

So it's better to mention that the operator installation may take a few seconds.

Version(s):
4.10+

Examples:
  * PR applies to all versions after a specific version (e.g. 4.8): 4.8+
  * PR applies to the in-development version (e.g. 4.11) and future versions: 4.11+
  * PR applies only to a specific single version (e.g. 4.10): 4.10
  * PR applies to multiple specific versions (e.g. 4.6-4.8): 4.6, 4.7, 4.8 --->

Issue:

Link to docs preview:

Additional information:

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@Rupesh-git-eng, I checked with the writer who works on this content, and he's added this clarification here: https://github.com/openshift/openshift-docs/pull/45516/files#diff-9a2ee215dd497eaa5b328d65fea3f26f823eb55a11f86e11699e94f24ab6c532R124

If this is acceptable, will you close your PR?

**Comment by Rupesh-git-eng** [comment]
Hi, 

It is better to put that under **". To verify that the Operator is installed, enter the following command:"**

File: modules/nw-metallb-installing-operator-cli.adoc [hunk 1]
```diff
@@ -108,7 +108,7 @@ spec:
 EOF
 ----
 
-. Confirm the install plan is in the namespace:
+. Confirm the install plan is in the namespace (Installation of operator may take a few seconds):
```
Comment (kalexand-rh) [inline modules/nw-metallb-installing-operator-cli.adoc:111]: @michaelryanpeter, is this kind of Operator disclaimer used elsewhere in the docs?

Comment (michaelryanpeter) [reply modules/nw-metallb-installing-operator-cli.adoc:111]: We don't use this type of disclaimer in the OLM docs. However, I have seen similar things in other places:
* https://docs.openshift.com/container-platform/4.14/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.html#distr-tracing-deploy-production_distr-tracing-jaeger-configuring (step four)
* https://docs.openshift.com/container-platform/4.14/logging/cluster-logging-upgrading.html#logging-upgrading-clo_cluster-logging-upgrading (Verification step one)

**Comment by kalexand-rh** [comment]
4.11 is now [EOL](https://access.redhat.com/support/policy/updates/openshift), and we are no longer updating it. Please set the versions to 4.12+.

**Pull request closed without merging**